eth/ethconfig/config.go

```diff
@@ -129,12 +129,13 @@ type Config struct {
 	// State scheme represents the scheme used to store ethereum states and trie
 	// nodes on top. It can be 'hash', 'path', or none which means use the scheme
 	// consistent with persistent state.
-	StateScheme          string                `toml:",omitempty"`
-	PathNodeBuffer       pathdb.NodeBufferType `toml:",omitempty"` // Type of trienodebuffer to cache trie nodes in disklayer
-	ProposeBlockInterval uint64                `toml:",omitempty"` // Keep the same with op-proposer propose block interval
-	EnableProofKeeper    bool                  `toml:",omitempty"` // Whether to enable proof keeper
-	KeepProofBlockSpan   uint64                `toml:",omitempty"` // Span block of keep proof
-	JournalFileEnabled   bool                  `toml:",omitempty"` // Whether the TrieJournal is stored using journal file
+	StateScheme             string                `toml:",omitempty"`
+	PathNodeBuffer          pathdb.NodeBufferType `toml:",omitempty"` // Type of trienodebuffer to cache trie nodes in disklayer
+	ProposeBlockInterval    uint64                `toml:",omitempty"` // Keep the same with op-proposer propose block interval
+	EnableProofKeeper       bool                  `toml:",omitempty"` // Whether to enable proof keeper
+	KeepProofBlockSpan      uint64                `toml:",omitempty"` // Span block of keep proof
+	JournalFileEnabled      bool                  `toml:",omitempty"` // Whether the TrieJournal is stored using journal file
+	EnableRecoverDiffLayers bool                  `toml:",omitempty"` // Whether enable recover diff layers for node buffer list.
```

Comment:
Only one change in L138, no change to other code lines. Pls keep them the same style.

Reply:
Auto changed by gofmt to keep aligning.